docs/web3-eth.rst

```diff
@@ -1600,7 +1600,7 @@ Executes a message call transaction, which is directly executed in the VM of the
 Parameters
 ----------
 
-1. ``Object`` - A transaction object see :ref:`web3.eth.sendTransaction <eth-sendtransaction-return>`, with the difference that for calls the ``from`` property is optional as well.
+1. ``Object`` - A transaction object, see :ref:`web3.eth.sendTransaction <eth-sendtransaction-return>`. For calls the ``from`` property is optional however it is highly recommended to explicitly set it or it may default to `address(0)` depending on your node or provider.
```

Comment:
👍 Perfect.

WDYT about adding a similar note for `method.call` here?
https://web3js.readthedocs.io/en/v1.2.6/web3-eth-contract.html#methods-mymethod-call

Reply:
good idea, just added in latest commit 👍